Luke7787/ImmaculateInventors
# Test/item services

Summary:

Made tests for the items and made it so that it uses mongo memory server instead of mockingoose.

How to test:

`npm install`

`npm run test`

The item tests should pass but the user tests probably won't pass since the code is still commented out
